Lesson19/Kadir/script.js

```diff
@@ -43,11 +55,50 @@ console.log(images);
 images[1].remove();
 
 // 5. Add a border to all images on the page (2px solid #272932).
+console.log(images);
+images[0].style.border="2px solid #272932";
+
 // 6. Change the text of the first heading on the page to "Welcome to my Website".
+const headingReplace = document.getElementById('title');
+console.log(headingReplace);
+headingReplace.innerHTML = "Welcome to my Website";
+
+
 // 7. Replace the first image on the page with a new image ("https://picsum.photos/400/200").
-// 8. Remove the first list item from the unordered list.
-// 9. Append a new list item to the end of the ordered list with the text "Last Item".
-// 10. Add a button (in html) with "Hide images" text. When a button is clicked, change the text of the button to "Show images" and hide all the images on the web page (set display to none).
+images[0].src="https://picsum.photos/400/200";
 
+// 8. Remove the first list item from the unordered list.
+const liItems = document.body.querySelectorAll('li');
+liItems[0].remove();
 
+// 9. Append a new list item to the end of the ordered list with the text "Last Item".
+    el = document.createElement('li');
+    el.innerHTML = "Last Item";
+    document.getElementById('lists').appendChild(el);
 
+// 10. Add a button (in html) with "Hide images" text. When a button is clicked, change the text of the button to "Show images" and hide all the images on the web page (set display to none).
+    newButton = document.createElement('button');
```

Comment:
```suggestion
    const newButton = document.createElement('button');
```

Reply:
Also, I'd say it makes more sense to add the button in HTML code as it should always stay on the screen.
Right now, the code does not work as expected for me. Did it work for your?